src/main/scala/com/softwaremill/mqperf/mq/PostgresMq.scala

```diff
@@ -0,0 +1,126 @@
+package com.softwaremill.mqperf.mq
+
+import java.time.Instant
+import java.util.UUID
+import java.util.concurrent.TimeoutException
+
+import cats.data.NonEmptyList
+import com.softwaremill.mqperf.config.TestConfig
+import doobie._
+import doobie.implicits._
+import doobie.postgres.implicits._
+import doobie.implicits.legacy.instant._
+import cats.implicits._
+import cats.effect._
+import doobie.hikari.HikariTransactor
+
+import scala.concurrent.duration._
+import doobie.util.log.{ExecFailure, ProcessingFailure, Success}
+
+import scala.concurrent.ExecutionContext
+
+class PostgresMq(testConfig: TestConfig) extends Mq {
+  override type MsgId = UUID
+  implicit val cs: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
+  implicit val timer: Timer[IO] = IO.timer(ExecutionContext.global)
```

Comment:
given latest discoveries about global it might be better to use a custom EC for both of these